docs/data-sources/service_accounts.md

```diff
@@ -23,7 +23,12 @@ data "twingate_service_accounts" "foo" {
 
 ### Optional
 
-- `name` (String) Filter results by the name of the Service Account.
```

Comment:
We should probably put some wording in here that if no options are passed it will return all service accounts. Also only one option can be used at a time.

Reply:
sure, added